/// fail: Break statement outside of loop

def main() {
    break
}
